bin/stack_functions

```diff
@@ -400,6 +400,11 @@ function validate_radius_varz {
 
 function test_varz_value {
     varz=$1
-    value=$( printf "%.0f" $(wget 0.0.0.0:8302 -O- | grep $varz | grep -v 'HELP\|TYPE' | awk '{print $2'}) )
+    if [ -z "$2" ]; then
+        path=8080/$2
```

Comment:
not a path -- it's port + path ?

How does this possibly work?  -z tests for the value being zero?  I think you need a negative test somewhere.

Reply:
Addressed.